examples/example_multi_genome.py

```diff
@@ -0,0 +1,138 @@
+"""
+Example for evolutionary regression with multiple genomes
+=========================================================
+
+Example demonstrating the use of Cartesian genetic programming with multiple
+genomes per individual for a simple regression task with a piecewise-defined
+target function.
+"""
+
+# The docopt str is added explicitly to ensure compatibility with
+# sphinx-gallery.
+docopt_str = """
+   Usage:
+     example_minimal.py [--max-generations=<N>]
+
+   Options:
+     -h --help
+     --max-generations=<N>  Maximum number of generations [default: 300]
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy.constants
+from docopt import docopt
+
+import cgp
+
+args = docopt(docopt_str)
+
+# %%
+# We first define a target function.  The function applies different
+# transformations to the input depending whether the input is less or greater
+# than zero. We thus need to fit two different functions.
+
+
+def f_target(x):
+    return np.select([x < 0, x >= 0], [-x, x ** 2 + 1.0])
+
+
+# %%
+# Then we define the objective function for the evolution. It uses the
+# mean-squared error between the output of the expression represented by a given
+# individual and the target function evaluated on a set of random points. We
+# here either evaluate the function represented by the first (``f[0]``) or the second
+# genome (``f[1]``), depending whether the input is less or greater than zero.
+
+
+def objective(individual):
+    if individual.fitness is not None:
+        return individual
+
+    n_function_evaluations = 1000
+
+    np.random.seed(1234)
+
+    f = individual.to_numpy()
```

Comment:
maybe add a comment that `f` is now a list? otherwise line 58 is quite confusing. or maybe add an additional comment there?

Reply:
Good idea, done.